src/main/java/com/dhbw/thesim/impexp/JsonHandler.java

```diff
@@ -55,6 +57,32 @@ public static void setDirectory() {
         }
     }
 
+    /**
+     * All json-files with "scenario" in their names are returned. (independent if capitalized or not)
+     * @return ArrayList<String> containing the proper file names
+     * @throws FileNotFoundException if the "workingDirectory" does not exist yet
+     */
+    public static ArrayList<String> getExistingScenarioFileNames() throws FileNotFoundException {
+        ArrayList<String> existingScenarioFiles = new ArrayList<>();
+
+        //set the directoryPath
+        File directoryPath = new File(workingDirectory);
+        if (!directoryPath.isDirectory()){
+            throw new FileNotFoundException("The directory: "+workingDirectory+" does not exist yet.");
+        }
+        //create a FilenameFilter that filters for all json-Files with scenario (independent if capitalized or not) in its names
+        FilenameFilter textFileFilter = new FilenameFilter(){
+            public boolean accept(File dir, String name) {
+                String lowercaseName = name.toLowerCase();
+                return lowercaseName.endsWith(".json") && lowercaseName.contains("scenario");
```

Comment:
Wenn nach "scenario" gefiltert wird, muss das auch in den Namen geforced werden.
Es muss also noch exportScenarioConfig angepasst werden, dass "scenario" z.B. am Ende vom Dateinamen eingefügt wird.

Reply:
Das hätte ich tatsächlich auf GUI ebene vermutet, dass das dann einfach angehängt weitergegeben wird. Und dem Nutzer zuvor aber auch visuell (?) entsprechend angedeutet wird